lib/pay_web/schema/content_types.ex

```diff
@@ -308,6 +308,7 @@ defmodule PayWeb.Schema.ContentTypes do
     value(:discover, as: "discover")
     value(:jcb, as: "jcb")
     value(:unionpay, as: "unionpay")
+    value(:maestro, as: "maestro")
```

Comment:
i didn't check, but does `maestro` also need to be added to our seeds?

Reply:
Thought of mapping it with Bambora brand return value Maestro.  Removed it from seed now.